src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -8,6 +8,9 @@
  */
 public class SqrtOperation {
     public double sqrt(double number) {
-        return 0.0;
+        if (number < 0) {
+            throw new IllegalArgumentException("Expected non-negative number, got? " + number);
```

Comment:
'?' what for?

Reply:
It was in the task condition, but I agree it is redundant. I will delete it.